docs/about.md

```diff
@@ -1,22 +1,14 @@
-# About Heptio Ark
+# How Ark Works
 
-Heptio Ark provides customizable degrees of recovery for all Kubernetes objects (Pods, Deployments, Jobs, Custom Resource Definitions, etc.), as well as for persistent volumes. This recovery can be cluster-wide, or fine-tuned according to object type, namespace, or labels.
+Each Ark operation -- on-demand backup, scheduled back, restore -- is a custom resource, defined with a Kubernetes [Custom Resource Definition (CRD)][20] and stored in [etcd][22]. The config custom resource specifies core information and customized options such as cloud provider settings. These resources are handled by [custom controllers][21] on the Kubernetes API server.
 
-Ark is ideal for the disaster recovery use case, as well as for snapshotting your application state, prior to performing system operations on your cluster (e.g. upgrades).
-
-## Features
-
-Ark provides the following operations:
-
-* On-demand backups
-* Scheduled backups
-* Restores
+Each controller watches its custom resource for API requests (Ark operations), performs validations, and handles the logic for interacting with the cloud provider API -- for example, managing object storage and persistent volumes.
```

Comment:
"handles the logic for interacting with the cloud provider API..." sounds a little weird to me, as if that's the only thing each controller does - that's just one of many things. I'd probably say something more general like "executes backups, restores, etc."

Reply:
It occurs to me that this sentence boils down to "The controller does what a controller does." Let's try ... getting rid of it altogether :-)